tests/letstest/README.md

```diff
@@ -15,30 +15,59 @@ simple aws testfarm scripts for certbot client testing
     are needed, they need to be requested via online webform.
 
 ## Usage
+  - To install the necessary dependencies on Ubuntu 16.04, run:
+```
+sudo apt install awscli python-yaml python-boto3 fabric
+```
+
   - Requires AWS IAM secrets to be set up with aws cli
   - Requires an AWS associated keyfile <keyname>.pem
 
 ```
->aws configure --profile HappyHacker
+>aws configure --profile <profile name>
 [interactive: enter secrets for IAM role]
->aws ec2 create-key-pair --profile HappyHacker --key-name MyKeyPair --query 'KeyMaterial' --output text > MyKeyPair.pem
+>aws ec2 create-key-pair --profile <profile name> --key-name <key name> --query 'KeyMaterial' --output text > whatever/path/you/want.pem
 ```
+Note: whatever you pick for `<key name>` will be shown to other users with AWS access.
+
+When prompted for a default region name, enter: `us-east-1`
+
 then:
 ```
->python multitester.py targets.yaml MyKeyPair.pem HappyHacker scripts/test_apache2.sh
+>python multitester.py targets.yaml /path/to/your/key.pem <profile name> scripts/<test to run>
+```
+
+You can only run up to two tests at once. The following error is often indicative of there being too many AWS instances running on our account:
+```
+NameError: name 'instances' is not defined
+```
+
+If you see this, you can run the following command to shut down all running instances:
+```
+aws ec2 terminate-instances --profile <profile name> --instance-ids $(aws ec2 describe-instances --profile <profile name> | grep <profile name> | cut -f8)
```

Comment:
Based upon your latest commit, I think this should be `... grep <key name> ...`

Reply:
fixed, thanks for catching this!